### PR TITLE
BZ1869798: Change message to have only IPI cluster

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Bug: https://bugzilla.redhat.com/show_bug.cgi?id=1869798

Releases: 4.7, 4.8, 4.9

@gpei - Kindly review this fix

Preview: https://deploy-preview-35218--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/configuring-private-cluster?utm_source=github&utm_campaign=bot_dp#private-clusters-setting-api-private_configuring-private-cluster

### CODE REVIEW

**Comment by gpei** [comment]
LGTM, thanks for the update.

File: modules/private-clusters-setting-api-private.adoc [hunk 1]
```diff
@@ -22,7 +22,14 @@ After you deploy a cluster to Amazon Web Services (AWS) or Microsoft Azure, you
 
 .. Delete the `api.$clustername.$yourdomain` DNS entry in the public zone.
 
-. From your terminal, list the cluster machines:
+. Remove the external load balancers:
++
+[IMPORTANT]
+====
+You can run the following steps only for Installer-Provisioned Infrastructure (IPI) cluster. For User-Provisioned Infrastructure (UPI) cluster, you must manually remove or disable the external load balancers.
```
Comment (jeana-redhat) [inline modules/private-clusters-setting-api-private.adoc:29]:
```suggestion
You can run the following steps only for an installer-provisioned infrastructure (IPI) cluster. For a user-provisioned infrastructure (UPI) cluster, you must manually remove or disable the external load balancers.
```

File: modules/private-clusters-setting-api-private.adoc [hunk 2]
```diff
@@ -43,16 +50,16 @@ lk4pj-worker-us-east-1b-zgpzg   running   m4.xlarge   us-east-1   us-east-1b   1
 +
 You modify the control plane machines, which contain `master` in the name, in the following step.
 
-. Remove the external load balancer from each control plane machine.
-.. Edit a control plane `Machine` object to remove the reference to the external load balancer.
+.. Remove the external load balancer from each control plane machine.
+... Edit a control plane `Machine` object to remove the reference to the external load balancer.
```
Comment (jeana-redhat) [inline modules/private-clusters-setting-api-private.adoc:54]:
```suggestion
... Edit a control plane `Machine` object to remove the reference to the external load balancer:
```
I know you didn't add this line, but would you mind fixing this while you're in here?

**Review by jeana-redhat — COMMENTED** [review]
Just a couple minor things too look at :)

**Pull request merged**